dataservice/api/genomic_file/resources.py

```diff
@@ -58,6 +61,11 @@ def get(self, filter_params, after, limit):
                  .join(Biospecimen.participant)
                  .filter(Participant.study_id == study_id))
 
+        from dataservice.api.read_group.models import ReadGroupGenomicFile
+        if read_group_id:
+            q = (q.join(ReadGroupGenomicFile)
+                 .filter(ReadGroupGenomicFile.read_group_id == read_group_id))
```

Comment:
I think this query still have the same problem what we have with biospecimen-diagnosis

Reply:
It doesn't, I tested it to make sure we're not getting back extra genomic files